docs/vercel/public/sakura.css

```diff
@@ -1,3 +1,4 @@
+/* $color-text: #dedce5; */
```

Comment:
Was this file manually edited, or did you grab one of the other stylesheets?

Reply:
I overrode with https://github.com/oxalorg/sakura/blob/master/css/sakura-dark.css

Reply:
I renamed the css file to `sakura-dark.css`